Improve wording on Broker configuration (#1904)
centreon/centreon-documentation: versioned_docs/version-22.04/service-mapping/remote-server.md

---
id: remote-server
title: Install on a remote server
---

The procedure for installing the Centreon BAM module on a Centreon remote server
is the same as on a central server and needs additionnal action.

> After you installed the Centreon BAM module using the standard installation procedure, please follow the additional steps described below.

## Configure Centreon Broker

> Configuring the Centreon BAM module on a Centreon remote server requires
> configuring the Centreon Broker from the Centreon central server. This allows
> the Centreon remote server to manage calculations on Centreon BAM. The
> configuration will be automatically sent to the Centreon remote server when you
> export your business activities to the Centreon remote server from the central

Reviewer: ```suggestion
The **Centreon BAM** module needs specific Broker outputs to be configured for the Centreon remote server.
```

server.

> To do so, you need to modify the Centreon Broker Master configuration of the

Reviewer: ```suggestion
To do so, you need to log in to your central server and modify the Centreon Broker Master configuration of the
```
